remme/tp/account.py

```diff
@@ -106,36 +121,40 @@ def _transfer_from_address(self, context, address_from, transfer_payload):
             raise InvalidTransaction('Could not transfer with zero amount.')
 
         if not transfer_payload.address_to.startswith(self._prefix) \
+                and not transfer_payload.address_to.startswith(NodeAccountHandler()._prefix) \
                 and transfer_payload.address_to != ZERO_ADDRESS:
             raise InvalidTransaction('Receiver address has to be of an account type.')
 
         if address_from == transfer_payload.address_to:
             raise InvalidTransaction('Account cannot send tokens to itself.')
 
-        signer_account, receiver_account = get_multiple_data(context, [
-            (address_from, Account),
-            (transfer_payload.address_to, Account),
+        sender_account_pb_class = Account if address_from[:6] == self._prefix else NodeAccount
```

Comment:
Basically this allows as to set any prefix different from `self._prefix`. That is pretty much enough to break the state.

Reply:
@eugene-babichenko, done.